geary-papermc-spawning/src/main/kotlin/com/mineinabyss/geary/papermc/spawning/tasks/SpreadSpawnTask.kt

```diff
@@ -0,0 +1,29 @@
+package com.mineinabyss.geary.papermc.spawning.tasks
+
+import com.github.shynixn.mccoroutine.bukkit.launch
+import com.mineinabyss.geary.papermc.gearyPaper
+import com.mineinabyss.geary.papermc.spawning.config.SpreadSpawnSectionsConfig
+import com.mineinabyss.geary.papermc.spawning.spread_spawn.SpreadSpawner
+import kotlinx.coroutines.delay
+import me.dvyy.sqlite.Database
+import org.bukkit.World
+import kotlin.time.Duration.Companion.seconds
+
+class SpreadSpawnTask(db: Database, world: World, configs: SpreadSpawnSectionsConfig) {
+    val spreadSpawner = SpreadSpawner(db, world, configs)
```

Comment:
Since you aren't using db or world here, it's worth just passing SpreadSpawner to your constructor, and doing all the setup stuff in the feature class